pyzo/tools/pyzoLogger.py

```diff
@@ -6,17 +6,40 @@
 
 
 import sys, os, code
-
 import pyzo
+from pyzo.util.qt import QtCore, QtGui, QtWidgets  # noqa
 from pyzo.core.shell import BaseShell
 from pyzo.core.pyzoLogging import splitConsole
 
-
 tool_name = pyzo.translate("pyzoLogger","Logger")
 tool_summary = "Logs messages, warnings and errors within Pyzo."
+
+
+class PyzoLogger(QtWidgets.QWidget):
```

Comment:
This wrapper class will cause problems, because in some places we use `pyzo.toolManager.getTool('pyzologger')` and do stuff with it. It it possible to fix the margins without wrapping the widget in another widget? Otherwise, this class will need a public property to get access to the shell, and all occurrences have to be updated.